Descent3/init.cpp

```diff
@@ -1548,40 +1548,6 @@ tTempFileInfo temp_file_wildcards[] = {{"d3s*.tmp"}, {"d3m*.tmp"}, {"d3o*.tmp"},
                                        {"d3c*.tmp"}, {"d3t*.tmp"}, {"d3i*.tmp"}};
 int num_temp_file_wildcards = sizeof(temp_file_wildcards) / sizeof(tTempFileInfo);
 
-// Returns true if Mercenary is installed
-bool MercInstalled() {
```

Comment:
instead of blowing this away, how about looking for `merc_hid > 0`? that'll support CD installs without the merc expansion.

Reply:
That's way more elegant. I think I'll do this all locally when I have time to keep from so many commits in the history.